### PR TITLE
S1 t5 search

### PR DESCRIPTION
**Pull request by JBHarrison5**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 //extracting info from database
-
 require_once 'functions.php';
 databaseSetUp();
 
+if (isset($_GET['search'])) {
+    $query = $db->prepare("SELECT `id`, `date`, `rating`, `review`, `image`, `flavour`, `location`
+                            FROM `mcflurrys`
+                            WHERE `date` LIKE CONCAT('%', :search, '%')
+                            OR `review` LIKE CONCAT('%', :seach, '%')
```
Comment (matt-ioa) [inline index.php]: You've got a typo

File: index.php [hunk 2]
```diff
@@ -13,12 +22,22 @@
...
+//check if search is set then scan multiple fields in the database to have a look
+if (isset($_GET['search'])) {
+    $query = $db->prepare("SELECT `id`, `date`, `rating`, `review`, `image`, `flavour`, `location`
+                            FROM `mcflurrys`
+                            WHERE `date` LIKE CONCAT('%', :search, '%')
+                            OR `review` LIKE CONCAT('%', :search , '%')
+                            OR `flavour` LIKE CONCAT('%', :search , '%')
+                            OR `location` LIKE CONCAT('%', :search , '%')
+                            OR `country` LIKE CONCAT('%', :search , '%');");
+    $query->bindParam(':search', $_GET['search']);
```
Comment (matt-ioa) [inline index.php:30]: Don't you have this exact same code above?

File: index.php [hunk 3]
```diff
@@ -51,22 +70,50 @@
...
-    foreach ($result as $item) {
-        echo '<div class="mcflurry">';
+
+    //covers the case where search doesn't return anything
+    if (count($result) === 0){
+        echo "<h3>No Results Found</h3>";
+    }
+    else {
+        //can probably refactor this. Maybe make a function to assign all nulls to unknown
+        //in the functions php file
```
Comment (matt-ioa) [inline index.php:77]: Good idea. If you don't do it now, it might never happen, so why not give it a go?

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Looks good, but some changes requested

File: index.php (hunk 3 above)
Comment (JBHarrison5) [reply index.php:77]: have done this now

File: information.php [hunk 4]
```diff
@@ -29,12 +29,25 @@
 </nav>
 <main>
     <?php
+//    echo '<div class="mcflurry information">';
+//        echo '<img src="images/' . $information['image'] . '.jpg">';
+//        echo '<h2>' . $information['flavour'] . '</h2>';
+//        echo '<h4>' . $information['location'] . ' on ' . $information['date'] . '</h4>';
+//        displayStars($information['rating']);
+//        echo '<p>' . $information['review'] . '</p>';
+//    echo '</div>';
```
Comment (matt-ioa) [inline information.php]: Get rid of this

**Review by matt-ioa — APPROVED** [review]

**Pull request merged**